project/DottySupport.scala

```diff
@@ -12,8 +12,8 @@ import sbt.librarymanagement.{
   * Settings to support validation of TastyUnpickler against the release of dotty with the matching TASTy version
   */
 object TastySupport {
-  val supportedTASTyRelease = "3.0.0-M2" // TASTy version 25.1
-  val scala3Compiler = "org.scala-lang" % "scala3-compiler_3.0.0-M2" % supportedTASTyRelease
+  val supportedTASTyRelease = "3.0.0-M3" // TASTy version 26.1
+  val scala3Compiler = "org.scala-lang" % "scala3-compiler_3.0.0-M3" % supportedTASTyRelease
 }
```

Comment:
We can also update the version bellow
```diff
object DottySupport {
-  val dottyVersion = "3.0.0-M2"
+  val dottyVersion = "3.0.0-M3"
```

Reply:
I think that is usually changed in a separate PR

Reply:
OK

Reply:
Any reason to do it in a separate PR?  In the absence of reasons let's update both in this one, IMO.

Reply:
alright will do